src/sentry/static/sentry/app/components/releaseStats.jsx

```diff
@@ -22,10 +22,11 @@ const ReleaseStats = React.createClass({
       return null;
     }
 
-    let releaseSummary = tct('[commitCount] commits by [authorCount] authors', {
-      commitCount,
-      authorCount
-    });
+    let releaseSummary = (
```

Comment:
i think you should maybe be using `tn` here, which handles pluralization.. although not sure if there's a way to handle multiple pluralizations in the same sentence. cc @mitsuhiko